pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,98 @@
+---
+date: '2024-04-17T15:45:00.000Z'
```

Comment:
Should we update this date right before the merge?

Reply:
yes we will update it tomorrow